config/config.exs

```diff
@@ -25,5 +25,15 @@ config :logger, :console,
 # of this file so it overrides the configuration defined above.
 import_config "#{Mix.env()}.exs"
 
+config :phoenix, :json_library, Jason
+
 config :turn_junebug_expressway, :turn,
   hmac_secret: System.get_env("TURN_HMAC_SECRET") || "REPLACE_ME"
+
+config :turn_junebug_expressway, :junebug,
+  from_addr: System.get_env("JUNEBUG_FROM_ADDR") || "+2712345",
+  transport_type: System.get_env("JUNEBUG_TRANSPORT_TYPE") || "telnet"
+
+config :turn_junebug_expressway, :rabbitmq,
+  messages_queue: System.get_env("MESSAGES_QUEUE") || "test_messages_queue",
+  urn: "amqp://guest:guest@localhost:5672/%2f"
```

Comment:
Does this not need to also be an env var?

Reply:
Yes!, will be fixed in next PR